### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by XhmikosR** [nlf/blankie#34]
```
C:\Users\xmr\Desktop\srihash.org>npm i
npm WARN deprecated @hapi/hoek@6.2.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).

C:\Users\xmr\Desktop\srihash.org>npm ls @hapi/hoek@6.2.4
srihash.org@0.2.0 C:\Users\xmr\Desktop\srihash.org
`-- blankie@4.1.1
  `-- @hapi/hoek@6.2.4
```

It seems `@hapi/hoek` is no longer supported and results in an npm warning. Not super important but it'd be nice if deps were updated :)

**Comment by nlf** [comment]
done and published as 5.0.0

**Comment by XhmikosR** [comment]
Thanks!
